Creating a new project is showing an error.

Hi,

I am running the following command from the documentation:
`npx create-mastra@latest --components agents,tools --llm openai --example`

But I am getting an error:

```
┌  Mastra Create
│
◇  What do you want to name your project?
│  mastra-agent
│
◇  Project created
│
◇  NPM dependencies installed
│
◇  mastra installed
│
◇  @mastra/core installed
│
◇  .gitignore added
│
└  Project created successfully


│
◇  An error occurred while initializing Mastra
TypeError: components.map is not a function or its return value is not iterable
    at init (file:///Users/marco/.npm/_npx/8cb5076d6146e2d0/node_modules/create-mastra/dist/index.js:1658:21)
    at async create (file:///Users/marco/.npm/_npx/8cb5076d6146e2d0/node_modules/create-mastra/dist/index.js:1768:3)
    at async Command.<anonymous> (file:///Users/marco/.npm/_npx/8cb5076d6146e2d0/node_modules/create-mastra/dist/index.js:1819:3)
│
└
   To start your project:

    cd mastra-agent
    npm run dev
```

Also I tried to run the npm run dev command, but I am getting this error:

```
> mastra-agent@1.0.0 dev
> mastra dev

(node:83862) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
INFO [2025-01-29 11:49:12.365 -0500] (DEPLOYER - Mastra Dev): Preparing .mastra directory
INFO [2025-01-29 11:49:12.367 -0500] (DEPLOYER - Mastra Dev): [Mastra Dev] - Ensuring your dependencies up to date...
up to date, audited 605 packages in 901ms
INFO [2025-01-29 11:49:13.341 -0500] (DEPLOYER - Mastra Dev): up to date, audited 605 packages in 901ms
89 packages are looking for funding
  run `npm fund` for details
INFO [2025-01-29 11:49:13.341 -0500] (DEPLOYER - Mastra Dev): 89 packages are looking for funding
  run `npm fund` for details
6 moderate severity vulnerabilities

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.
INFO [2025-01-29 11:49:13.343 -0500] (DEPLOYER - Mastra Dev): 6 moderate severity vulnerabilities

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.
INFO [2025-01-29 11:49:13.366 -0500] (Mastra CLI): [Mastra Dev] - Starting server...
node:internal/modules/esm/resolve:257
    throw new ERR_MODULE_NOT_FOUND(
          ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/marco/nanoservice/mastra-agent/.mastra/mastra.mjs' imported from /Users/marco/nanoservice/mastra-agent/.mastra/index.mjs
    at finalizeResolution (node:internal/modules/esm/resolve:257:11)
    at moduleResolve (node:internal/modules/esm/resolve:913:10)
    at defaultResolve (node:internal/modules/esm/resolve:1037:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:650:12)
    at #cachedDefaultResolve (node:internal/modules/esm/loader:599:25)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:582:38)
    at ModuleLoader.getModuleJobForImport (node:internal/modules/esm/loader:241:38)
    at ModuleJob._link (node:internal/modules/esm/module_job:132:49) {
  code: 'ERR_MODULE_NOT_FOUND',
  url: 'file:///Users/marco/nanoservice/mastra-agent/.mastra/mastra.mjs'
}

Node.js v22.11.0
ERROR [2025-01-29 11:49:15.878 -0500] (Mastra CLI): Server failed to start with error:
    message: null
^CINFO [2025-01-29 11:49:27.748 -0500] (Mastra CLI): [Mastra Dev] - Stopping server...
```

--- a/.changeset/odd-kangaroos-wink.md
+++ b/.changeset/odd-kangaroos-wink.md
@@ -1,0 +1,6 @@
+---
+'mastra': patch
+'create-mastra': patch
+---
+
+Fix cli create not parsing components flag

--- a/packages/cli/src/index.ts
+++ b/packages/cli/src/index.ts
@@ -61,9 +61,10 @@ program
             llmProvider: 'openai',
             addExample: false,
           });
+          return;
         }
         await create({
-          components: args.components,
+          components: args.components ? args.components.split(',') : [],
           llmProvider: args.llm,
           addExample: args.example,
           llmApiKey: args['llm-api-key'],

--- a/packages/create-mastra/src/index.ts
+++ b/packages/create-mastra/src/index.ts
@@ -47,7 +47,7 @@ program
     }
 
     await create({
-      components: args.components,
+      components: args.components ? args.components.split(',') : [],
       llmProvider: args.llm,
       addExample: args.example,
       llmApiKey: args['llm-api-key'],

